authnid/root.py

```diff
@@ -0,0 +1,37 @@
+from flask import Blueprint, request, redirect, render_template
+from flask import current_app as app
+from .utils import parse_sdn
+
+root = Blueprint('home', __name__)
+
+@root.route('/')
+def log_in_user():
+    # FIXME: Find or create user based on the X-Ssl-Client-S-Dn header
+    # TODO: Store/log the X-Ssl-Client-Cert in case it's needed?
+    if request.environ.get('HTTP_X_SSL_CLIENT_VERIFY') == 'SUCCESS' and is_valid_certificate(request):
+        sdn = request.environ.get('HTTP_X_SSL_CLIENT_S_DN')
+        # TODO: error handling for bad SDN, database errors, etc
```

Comment:
This is fine for now, but let's make sure we're tracking in PT as well.

Reply:
sure: https://www.pivotaltracker.com/story/show/158725026